### ISSUE TITLE
replace \n with whitespace in shortenMessage()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#99]
```
pretty obvious ;) greetings!


SmsCmd.java
shortenMessage(String message) {
  ...
  //shortendMessage = message.substring(0, shortenTo) + "...";
  shortendMessage = message.replace("\n", " ").substring(0, shortenTo) + "...";
  ...
}
```

Original issue reported on code.google.com by `ThDietri...@gmail.com` on 11 Feb 2011 at 2:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 Feb 2011 at 3:48
- Changed title: **replace \n with whitespace in shortenMessage()**
- Changed state: **WaitingForRelease**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
